src/features/events/components/EventTypeAutocomplete.tsx

```diff
@@ -55,13 +55,13 @@ const EventTypeAutocomplete: FC<EventTypeAutocompleteProps> = ({
   onFocus,
   showBorder,
   types,
-  typesModel,
   value,
 }) => {
   const [createdType, setCreatedType] = useState<string>('');
-
+  const { orgId } = useNumericRouteParams();
```

Comment:
I'm not entirely sure how I feel about this. I understand that it might feel like a small change and very convenient, but it changes the division of responsibility of the code significantly in a way that may not be apparent.

Before the change to this file, the orgId came with the model, i.e. it was being passed to this component by the user of this component, putting the user of the component in charge of what organization to retrieve the types from.

After this change, the orgId comes from the router (via `useNumericRouteParams()`), i.e. from the URL. This means that whoever uses this component is no longer able to control what organization it should be retrieving values from.

I don't think this is a big deal right now, but it does mean that if we introduce any UI (or modification to existing UI) where the user can modify events from different organizations side by side, this component will be useless there, because it will always work with the organization from the URL, and it won't be possible for the user of the component to change that without changing the code (again).